pipelines/run-pipelines-on-kubeflow-pipelines/README.md

```diff
@@ -0,0 +1,303 @@
+
+The following tutorials cover generic pipelines:
+- [Introduction to generic pipelines](../introduction-to-generic-pipelines)
+- [Run generic pipelines on Kubeflow Pipelines](../run-generic-pipelines-on-kubeflow-pipelines)
+- [Run generic pipelines on Apache Airflow](../run-generic-pipelines-on-apache-airflow)
+
+A _runtime specific_ pipeline comprises nodes that are implemented using generic components or _custom components_. Custom components are runtime specific and user-provided.
+
+In this intermediate tutorial you will learn how to add [Kubeflow Pipelines components](https://www.kubeflow.org/docs/components/pipelines/sdk/component-development/) to Elyra and how to utilize them in pipelines.
+
+![The completed tutorial pipeline](doc/images/completed-tutorial-pipeline.png)
+
+The features described in this tutorial require Elyra v3.2.0. The tutorial instructions were last updated using Elyra v3.2.0 and Kubeflow v1.3.0.
+
+> Elyra v.3.2.0 does not support [Kubeflow Pipelines Python function-based components](https://www.kubeflow.org/docs/components/pipelines/sdk/python-function-components/).
+
+### Prerequisites
+
+- [JupyterLab 3.x with the Elyra extension v3.2 (or newer) installed](https://elyra.readthedocs.io/en/stable/getting_started/installation.html).
+- Access to a [local](https://elyra.readthedocs.io/en/stable/recipes/deploying-kubeflow-locally-for-dev.html) or [cloud](https://www.kubeflow.org/docs/started/installing-kubeflow/) Kubeflow Pipelines deployment.
+
+Some familiarity with Kubeflow Pipelines and Kubeflow Pipeline components is required to complete the tutorial. If you are new to Elyra, please review the [_Run generic pipelines on Kubeflow Pipelines_](../run-generic-pipelines-on-kubeflow-pipelines) tutorial. It introduces concepts and tasks that are used in this tutorial, but not explained here to avoid content duplication.
+
+#### Information to collect before starting
+
+Collect the following information for your Kubeflow Pipelines installation:
+- API endpoint, e.g. `http://kubernetes-service.ibm.com/pipeline`
+- Namespace, for a multi-user, auth-enabled Kubeflow installation, e.g. `mynamespace`
+- Username, for a multi-user, auth-enabled Kubeflow installation, e.g. `jdoe`
+- Password, for a multi-user, auth-enabled Kubeflow installation, e.g. `passw0rd`
+- Workflow engine type, which should be `Argo` or `Tekton`. Contact your administrator if you are unsure which engine your deployment utilizes.
+
+Elyra utilizes S3-compatible cloud storage to make data available to notebooks and scripts while they are executed. Any kind of cloud storage should work (e.g. IBM Cloud Object Storage or Minio) as long as it can be accessed from the machine where JupyterLab is running and from the Kubeflow Pipelines cluster. Collect the following information:
```

Comment:
```suggestion
Elyra utilizes S3-compatible cloud storage to make data available to notebooks and scripts while they are executed. Any kind of S3-based cloud storage should work (e.g. IBM Cloud Object Storage or Minio) as long as it can be accessed from the machine where JupyterLab/Elyra is running and from the Kubeflow Pipelines cluster. Collect the following information:
```